app/controllers/contact_requests_controller.rb

```diff
@@ -1,26 +0,0 @@
-class ContactRequestsController < ApplicationController
```

Comment:
why is that one deleted?

Reply:
Because we reverted our merge.

Reply:
But why was is reverted? And even more importantly, why in this PR and not in a dedicated one?